docs/src/pages/landing-page/Index.vue

```diff
@@ -138,63 +141,62 @@
       <twitter-showcase-cards />
     </div>
 
-    <div class="window-height">
+    <div class="window-height support-quasar-section">
       <div class="row justify-center">
         <div class="col-6 col-xs-10">
-          <div class="lp-heading--large">
+          <div class="lp-heading--large support-quasar-section__title">
             Support quasar: Become sponsor!
           </div>
 
           <div class="lp-heading--small text-left support-quasar-caption-text">
             Working for a company or freelancer? You can contribute, ever a bit and getting something back.
           </div>
 
-          <div class="q-my-md lp-heading--quote">
+          <div class="q-my-md lp-heading--quote lp-my--medium">
             <q>So, um, we think we should discuss the bonus situation</q>
           </div>
 
           <q-btn
             color="lp-accent"
             label="Consult tiers"
             padding="sm"
-            class="text-size-14 text-bold shadow-bottom-small"
+            class="btn--call-to-action"
           />
         </div>
       </div>
 
       <div class="astronaut-hand">
         <q-img class="astronaut-hand--left" width="50%"  src="~assets/landing-page/homepage-background-images/astronaut-left-hand.png" />
-        <q-img class="astronaut-hand--right" width="50%" src="~assets/landing-page/homepage-background-images/astronaut-right-hand.png" />
+        <q-img class="astronaut-hand--right" width="50%" :img-style="{ width: '102%' }" src="~assets/landing-page/homepage-background-images/astronaut-right-hand.png" />
       </div>
     </div>
 
-    <div class="text-center lp-mb--large">
+    <div class="text-center lp-mb--large sponsors-section" id="sponsors_section">
       <q-icon size="xl" name="img:homepage-icons/medal.svg" />
       <div class="lp-heading lp-heading--large">Our Sponsors</div>
       <div class="lp-heading lp-heading--small">Every space odyssey has its patrons</div>
       <div class="row justify-center">
         <div class="col-8 text-size-12">
           <div class="q-my-md">Platinum Sponsors</div>
           <q-img :src="`sponsor-logos/${src}`" width="200px" v-for="(src, index) in sponsorLogos.platinum" :key="index"/>
-          <div class="q-my-md">Gold Sponsors</div>
-          <q-img :src="`sponsor-logos/${src}`" width="200px" v-for="(src, index) in sponsorLogos.gold" :key="index" />
+<!--          <div class="q-my-md">Gold Sponsors</div>-->
```

Comment:
Technically, if you don't provide any item, this wouldn't render anything anyway, and it's better to avoid leaving dead code around, especially in templates